perfkitbenchmarker/benchmark_sets.py

```diff
@@ -190,7 +191,7 @@ def GetBenchmarksFromFlags():
     else:
       benchmark_names.add(benchmark)
 
-  # Expand recursive sets
+# Expand recursive sets
```

Comment:
Please align comments to match the rest of the code (applies to the other file as well).
```
# Comment
x = 1
if condition:
  # Comment
  x = 2
```